internal/pkg/mapper/template/replacekeys/replace.go

```diff
@@ -1,144 +0,0 @@
-package replacekeys
-
-import (
-	"fmt"
-	"reflect"
-	"regexp"
-	"strings"
```

Comment:
Balicek `replacekeys` -> som premenoval na `replacevalues`, kedze to viac sedi.
A upravil som interface, ako sa to pouziva.
Kod sa prakticky nezmenil, ... oddelil som validaciu do samostatnej metody, spravil metody public ...

Povodne testy zostali.
Odporucam side-by-side diff, ... GitHub to zobrazuje dost divne.